contrib/mesos/docs/issues.md

```diff
@@ -37,27 +121,93 @@ Kubernetes clusters assume a custom Docker networking configuration that assigns
+This workaround may be mitigated down the road by:
+- Future support for IPv6 addressing in Docker and Kubernetes
+- Native IP-per-container support via Mesos with a custom Kubernetes network plugin
 
-### Orphan Pods
+### Scheduling
 
-The default `executor_shutdown_grace_period` of a Mesos slave is 3 seconds.
-When the executor is shut down it forcefully terminates the Docker containers that it manages.
-However, if terminating the Docker containers takes longer than the `executor_shutdown_grace_period` then some containers may not get a termination signal at all.
-A consequence of this is that some pod containers, previously managed by the framework's executor, will remain running on the slave indefinitely.
+Statements in this section regarding the "scheduler" pertain specifically to the Kubernetes-Mesos scheduler, unless otherwise noted.
 
-There are two work-arounds to this problem:
-* Restart the framework and it should terminate the orphaned tasks.
-* Adjust the value of `executor_shutdown_grace_period` to something greater than 3 seconds.
+Some factors that influence when pods are scheduled by k8s-mesos:
+- availability of a resource offer that "fits" the pod (mesos master/slave);
+- scheduler *backoff* (to avoid busy-looping) during pod scheduling (k8s-mesos scheduler)
+
+We have tried to mitigate the second item by cancelling the backoff period if an offer arrives that fits a pod-in-waiting.
+However, there is nothing that the scheduler can do if there are no resources available in the cluster.
+
+That said, the current scheduling algorithm is somewhat naive: it makes **no attempts to pack multiple pods into a single offer**.
+This means that each pod launch requires an independent offer.
+In a small cluster resource offers do not arrive very frequently.
+In a large cluster with a "decent" amount of free resources the arrival rate of offers is expected to be much higher.
+
+The slave on each host announces offers to Mesos periodically.
+In a single node cluster only a single slave process is advertising resources to the master.
+The master will pass those along to the scheduler, at some interval and level of 'fairness' determined by mesos.
+That scheduler will pair each resource offer with a pod that needs to be placed in the cluster.
+Once paired, a task is launched to instantiate the pod.
+The used resources will be marked as consumed, the remaining resources are "returned" to the cluster and the scheduler will wait for the next resource offer from the master... and the cycle repeats itself.
+This likely limits the scheduling throughput observable in a single-node cluster.
+
+We are currently running some benchmarks on the scheduling algorithm to establish some baselines, and are definitely thinking about ways to increase scheduling throughput- including scheduling multiple pods per offer.
+
+#### Runtime Configuration
+
+- mesos: `--offer_timeout` : Duration of time before an offer is rescinded from a framework.
```

Comment:
`--offer_timeout` is correct (see http://mesos.apache.org/documentation/latest/configuration/)